### PR TITLE
Cleanup why section

### LINKED ISSUE #55: Proposed changes to the narrative
1. Reorder sections -> Install > Sample > Why > Features 
2. Change the Why section so it talks about where does slog shine vs where does Uber library fall short
> from: We wanted tighter integration with stdlib's testing package
> to: Tight integration with stdlib's testing package
3. Consider merge the Why and Features section into one. 
4. Users section, consider rephrasing
If your company or project is using slog, feel free to open an issue or PR to amend this list.
vs
Let others know you're using slog, open an issue or PR to amend this list

Hope this helps

DavidS

### PR DESCRIPTION
**Pull request by nhooyr**
Closes #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,52 +73,47 @@ Here is a list of reasons how we improved on zap with slog.
    - We wanted an API that only accepted the equivalent of [zap.Any](https://godoc.org/go.uber.org/zap#Any) for every field.
      This is [slog.F](https://godoc.org/cdr.dev/slog#F).
 
-1. `slog` uses a very
+1. `sloggers/sloghuman` uses a very human readable format
```
Comment (coadler) [inline README.md]: I think this sounds a bit better:

```suggestion
1. [`sloghuman`](https://github.com/cdr/slog/tree/master/sloggers/sloghuman) provides a very human readable format
```

File: README.md [hunk 2]
```diff
@@ -73,52 +73,47 @@ Here is a list of reasons how we improved on zap with slog.
...
+   - slog colors distinct parts of each line to make it easier to scan logs. Even the JSON that represents
+     the fields in each log is syntax highlighted so that is very easy to scan. See the screenshot above.
+     - zap lacks appropriate colors for different levels and fields
+   - slog automatically prints one multiline field after the log to make errors and such much easier to read.
+     slog also automatically prints a Go 1.13 error chain as an array. See screenshot above.
+     - zap logs multiline fields and errors stack traces as JSON strings which made them unreadable in a terminal.
 
-1. Lack of [context.Context](https://blog.golang.org/context) support.
+1. Full [context.Context](https://blog.golang.org/context) support
+   `slog` lets you set fields in a `context.Context` such that any log with the context prints those fields.
```
Comment (coadler) [inline README.md]: This is a bit weird because they end up on the same line.

Should the second line be a bullet point?

File: README.md [hunk 3]
```diff
@@ -73,52 +73,47 @@ Here is a list of reasons how we improved on zap with slog.
...
+     [ObjectMarshaler](https://godoc.org/go.uber.org/zap/zapcore#ObjectMarshaler) to log Go structures. This was
+     very verbose and most of the time we ended up only implementing `fmt.Stringer` and using `zap.Stringer`
+     instead.
 
-1. We had many helper functions for logging but we wanted the line reported to be of the parent function.
-   zap has an [API](https://godoc.org/go.uber.org/zap#AddCallerSkip) for this but it's verbose and requires
-   passing the logger around explicitly.
+1. slog takes inspriation from Go's stdlib and implements [`slog.Helper`](https://godoc.org/cdr.dev/slog#Helper) which works just like
+   [`t.Helper`](https://golang.org/pkg/testing/#T.Helper) to mark the calling function as a helper and skip it when
+   reporting location info.
```
Comment (coadler) [inline README.md]: This is a pretty long bullet point. Maybe just leave this as
```
slog takes inspiration from Go's stdlib
```
and move the rest to bullet points?

Comment (nhooyr) [reply README.md]: Will fix.

File: README.md (hunk 2 above)
Comment (nhooyr) [reply README.md]: Will fix.

File: README.md (hunk 1 above)
Comment (nhooyr) [reply README.md]: Will fix.

File: README.md [hunk 4]
```diff
@@ -73,52 +73,47 @@ Here is a list of reasons how we improved on zap with slog.
...
    - zap has [zaptest](https://godoc.org/go.uber.org/zap/zaptest) but the API surface is large and doesn't
-     integrate well. It has no support for failing on any ERROR logs nor does it have a stateless API
-     that logs directly to a [`testing.TB`](https://golang.org/pkg/testing/#TB) without creating a
-     intermediary logger.
-
-## Users
-
-If your company or project is using slog, feel free to open an issue or PR to amend this list.
-
-- [Coder](https://github.com/cdr)
```
Comment (davidsalgado) [inline README.md]: I liked the idea, did you remove completely for any specific reason?

Comment (nhooyr) [reply README.md]: We talked on slack.

**Pull request merged**